ledger-core/v2/deploy/kube/ingress-nginx.yaml

```diff
@@ -0,0 +1,615 @@
+
```

Comment:
Is this file autogenerated?

Reply:
No, it's fixed and made some changes to make it work on actions